src/api/v1beta1/properties.go

```diff
@@ -130,6 +130,14 @@ func (dk *DynaKube) StatsdImage() string {
 	return resolveImagePath(newStatsdImagePath(dk))
 }
 
+func (dk *DynaKube) NeedsReadOnlyOneAgents() bool {
+	inSupportedMode := dk.HostMonitoringMode() || dk.CloudNativeFullstackMode()
+	if inSupportedMode && !dk.FeatureDisableReadOnlyOneAgent() {
+		return true
+	}
+	return false
```

Comment:
```suggestion
return (dk.HostMonitoringMode() || dk.CloudNativeFullstackMode())
     && !dk.FeatureDisableReadOnlyOneAgent()
```

Reply:
less lines != better

Reply:
(I think my version is clearer 😛 )

Reply:
I don't like `return true` and `return false` when you can just return the expression.
you can also do:
```go
inSupportedMode := dk.HostMonitoringMode() || dk.CloudNativeFullstackMode()
return inSupportedMode && !dk.FeatureDisableReadOnlyOneAgent()
```